### PR TITLE
Integrated map

### PR DESCRIPTION
**Pull request by mwjones2186**

### CODE REVIEW

**Review by JereamySmothers — APPROVED** [review]

File: models/User.js [hunk 1]
```diff
@@ -31,6 +31,10 @@ User.init(
         isEmail: true,
       },
     },
+    zipcode: {
+      type: DataTypes.STRING,
```
Comment (slorreina369) [inline models/User.js:35]: Should this be an integer? can you elaborate on why we're choosing to have it be a string?

File: controllers/api/user-routes.js [hunk 2]
```diff
@@ -49,16 +49,13 @@ router.get('/:id', (req, res) => {
 
 router.post('/', (req, res) => {
   // expects {username: 'Username', email: 'example@gmail.com', password: 'password1234'}
-  User.create({
-    username: req.body.username,
-    email: req.body.email,
-    password: req.body.password,
-  })
+  User.create(req.body)
```
Comment (slorreina369) [inline controllers/api/user-routes.js:52]: Can you elaborate on why this change was made?

Comment (mwjones2186) [reply controllers/api/user-routes.js:52]: because its the proper way the code should be written. Everything else is redundant

File: models/User.js (hunk 1 above)
Comment (mwjones2186) [reply models/User.js:35]: no

Comment (slorreina369) [reply models/User.js:35]: Can you elaborate on why we're choosing to have it be a string???

File: controllers/api/user-routes.js (hunk 2 above)
Comment (slorreina369) [reply controllers/api/user-routes.js:52]: Generally the industry standard is to explicity white list columns and fields, instead of automatically passing all the data. This is a potentional security risk if the code changes in the future and if we have a column that we don't want the user to be set or override(i.e. like an administrator flag). This also is less clear for future coders to understand. The way it was previously written was how we were taught and was industry best practice

**Review by slorreina369 — APPROVED** [review]
Code was pulled onto a seperate branch and rebased onto develop. Code appears to work and is functional. Previous questions about changes were not answered.
